mscore/importmxmlpass2.cpp

```diff
@@ -1755,7 +1755,7 @@ static void handleBeamAndStemDir(ChordRest* cr, const Beam::Mode bm, const Direc
             static_cast<Chord*>(cr)->setStemDirection(sd);
             cr->setBeamMode(Beam::Mode::AUTO);
             }
-      // terminate the currect beam and add to the score
+      // terminate the correct beam and add to the score
```

Comment:
wasn't sure if this was `current` or `correct`, I chose the latter.

Reply:
I believe it shgould be `current` though

Reply:
Done. f50dbab